oak_functions/proto/abi.proto

```diff
@@ -39,12 +39,14 @@ enum OakStatus {
   UNSPECIFIED = 0;
   // Success.
   OK = 1;
-  // Arguments invalid.
+  // Arguments invalid, e.g., deserializing the request failed.
   ERR_INVALID_ARGS = 2;
-  // Internal error.
+  // Internal error, e.g., when serialzing the response failed.
   ERR_INTERNAL = 3;
   // Error when the given number is not a valid handle.
   ERR_INVALID_HANDLE = 4;
+  // Error when serialization fails.
+  ERR_SERIALIZING = 5;
```

Comment:
Now this one looks off, since the other errors that are not named after serialization do mention serialization as an example, but this one not.

Reply:
Very good point, fixed!